### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by juniorbonner** [Audiveris/audiveris#618] [Q&A]
[Smooth Operator LH Piano.pdf](https://github.com/Audiveris/audiveris/files/10701679/Smooth.Operator.LH.Piano.pdf)

I am no doubt doing something obviously wrong, but I could do with some help with figuring out how this software works.
I have this pdf which I process through the software as per the guidance in the handbook and I just get a load of nonsense mxl file.
Is there something wrong with the pdf I am using?

**Comment by potero84** [comment]
Your PDF has very low resolution (74 ppi when 300 ppi is highly recommended for better performance) but the key thing in this case is that **the staff and bar lines are too light** and are lost in BINARY step (converting the image to black and white). It is hard to find music when you don't have staff lines or bar lines, even for a human been.
